data-es6.js

```diff
@@ -5948,6 +5979,7 @@ exports.tests = [
       */},
       res: {
         babel:       { val: false, note_id: "name-configurable", },
+        typescript:  typescript.corejs,
```

Comment:
This and some other tests from this category will work in environment w/o native `Function#name`, but not in actual environments with native. Run tests also in Chrome.